test/unit_tests/modules/nav-primary-spec.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+var chai = require( 'chai' );
+var sinon = require( 'sinon' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+
+describe( 'Get Event States', function() {
+  var $, navPrimary, sandbox;
+
+  jsdom();
+
+  before( function() {
+    $ = require( 'jquery' );
+    navPrimary = require( '../../../src/static/js/modules/nav-primary.js' );
+    sandbox = sinon.sandbox.create();
+  } );
+
+  beforeEach( function() {
+    //  Adding a simplified version of the thing we want to test.
+    //  Then calling the jQuery to test after
+    $( 'body' ).html(
+      '<div class="js-primary-nav">' +
+        '<div class="js-primary-nav_trigger"></div>' +
+        '<div class="nav-item-1">' +
+          '<a class="nav-link-1 js-primary-nav_link" ' +
+             'href="http:// github.com">Link</a>' +
+          '<div class="sub-nav-1 js-sub-nav" aria-expanded="false">' +
+            '<button class="sub-nav-back-1 js-sub-nav_back">Back</button>' +
+          '</div>' +
+        '</div>' +
+        '<div class="nav-item-2">' +
+          '<a class="nav-link-2 js-primary-nav_link" ' +
+             'href="http:// github.com">Link</a>' +
+          '<div class="sub-nav-2 js-sub-nav" aria-expanded="false">' +
+            '<button class="js-sub-nav_back">Back</button>' +
+          '</div>' +
+        '</div>' +
+      '</div>'
+    );
+    navPrimary.init();
+  } );
+
+  afterEach( function() {
+    sandbox.restore();
+  } );
+
+  describe( 'Primary Nav Link Events - Default state', function() {
+
+    it( 'should not navigate away from the page', function() {
+      $( '.nav-link-1' ).trigger( 'click' );
+      expect( window.location.href ).to.not.equal( 'http:// www.google.com/' );
+    } );
+
+    it( 'should toggle the sibling sub nav', function() {
+      // var toggleSiblingSpy = sandbox.spy( es.set.toggleExpandedState );
```

Comment:
I want to get these working next sprint, but I didn't want to forget where I left off. I can probably just stash them though.